saskatoon/fixtures/member-authuser.json

```diff
@@ -0,0 +1,18 @@
+[
+{
+  "model": "member.authuser",
+  "pk": 2,
+  "fields": {
+    "password": "pbkdf2_sha256$260000$B4izrSL5cMDhpz76Ml8oAd$y+vALyJB3hKoOGYietAefzmZQMA0TSPrbGq4WLI7yvs=",
```

Comment:
We should make sure the production instance doesn't use the same password as the one this sha256 represents.

Reply:
Yes for sure! I created a new password that I will send you.
Other thing is if someone else deploys this fixture they must be aware we could log into their admin panel unless they remove this superuser. So maybe having a superuser as initial deployment is not the best idea.
We could also keep this auth.user but remove its superuser status?

Reply:
Yes, no superuser is a wiser choice.

Reply:
Ok I'll fix this tomorrow. Keeping at least one auth.user is necessary to have a pick leader in harvest though. I'll create a new Person called "Pick Leader" as an example and remove Super User

Reply:
I removed the superuser and only kept one auth.user as pick leader.